onebusaway-android/src/main/java/org/onebusaway/android/ui/ReportStopProblemFragment.java

```diff
@@ -110,16 +111,31 @@ public void onSaveInstanceState(Bundle outState) {
     }
 
     @Override
+    protected Boolean isReportArgumentsValid() {
```

Comment:
Np.  My personal preference is to only use objects instead of primitives if we need to check for a null value, otherwise just stick to primitives.